unreleased.md

```diff
@@ -3,3 +3,7 @@
 **Added:**
 - react-native-bpk-component-pagination-dots:
   - Introducing the React Native pagination dots component.
+
+**Changed:**
```

Comment:
I don't think this warrants a changelog entry now. Nothing has changed from a consumer POV.

Reply:
We've deprecated the old keys and intend to remove them. That's definitely something we should communicate IMO